node-commons/src/assembly/resources/conf/iotdb-common.properties

```diff
@@ -952,6 +952,11 @@
 # schema_region_ratis_preserve_logs_num_when_purge=1000
 # data_region_ratis_preserve_logs_num_when_purge=1000
 
+# Raft Log disk size control
+# config_node_ratis_log_max_size = 2147483648
```

Comment:
Please make them consistent in different places.

Reply:
Done. Thanks for catching!